tests/gtest/avifrgbtoyuvtest.cc

```diff
@@ -808,14 +811,15 @@ INSTANTIATE_TEST_SUITE_P(
             /*yuv_depth=*/Values(8, 10),
             /*width=*/Values(1, 2, 127, 200),
             /*height=*/Values(1, 2, 127, 200),
-            Values(AVIF_RGB_FORMAT_RGB, AVIF_RGB_FORMAT_ARGB),
+            Values(AVIF_RGB_FORMAT_RGB, AVIF_RGB_FORMAT_RGBA),
             Range(AVIF_PIXEL_FORMAT_YUV444, AVIF_PIXEL_FORMAT_COUNT),
             // Test an odd and even number for threads. Not adding all possible
             // thread values to keep the number of test instances low.
             /*threads=*/Values(2, 7),
             /*avoidLibYUV=*/Values(true, false),
             Values(AVIF_CHROMA_UPSAMPLING_FASTEST,
-                   AVIF_CHROMA_UPSAMPLING_BILINEAR)));
+                   AVIF_CHROMA_UPSAMPLING_BILINEAR),
+            /*has_alpha=*/::testing::Bool()));
```

Comment:
I usually avoid `::testing::Bool()` because it is harder to understand than `Values(false, true)` and it is less convenient when debugging (when you just want to set one value and not both).